### ISSUE TITLE
VoiceOver is unable to access "Updated n minutes ago"

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [Ranchero-Software/NetNewsWire#2245]
Environment:
iOS 13.5.1
NNW 5.0.2

Steps:
1. Turn on VoiceOver
2. Touch the settings button at the bottom left corner to move VoiceOver focus there
3. Swipe right

Expected result:
VoiceOver should be able to navigate to the "Updated n minutes ago" text

Actual result:
VoiceOver skips this element.

**Comment by MarcoZehe** [comment]
This is also the case on iPad.

**Comment by vincode-io** [comment]
We've been having some problems getting VoiceOver to work on our custom views that are in the Navigation or Toolbars. We'll keep trying to figure it out.

**Comment by vincode-io** [comment]
@rizwankce Did we ever figure out what was going on here?

**Comment by rizwankce** [comment]
@vincode-io not really. I have hard time to get it working. One last thing I need to try out is create a separate app and test it out.
